### DISCUSSION TITLE
Letzten Dimmer-Wert wiederherstellen

### DISCUSSION THREAD

**Discussion by chrostek** [SukramJ/aiohomematic#1636] [Ideas]
Im Grunde dasselbe wie in #736 - die Discussion ist aber locked. Bei mir geht es um den HmIPW-DRD3

Laut:
https://community.home-assistant.io/t/setting-homematic-dimmers-to-last-dim-value/549920
und
https://homematic-forum.de/forum/viewtopic.php?t=21642

scheint es doch eine Möglichkeit zu geben. Nämlich das Senden von "OLD_LEVEL" als Value. Wäre es denkbar, das zu implementieren?

**Comment by SukramJ** [comment]
Die HmIP/HmIPW haben keinen `OLD_LEVEL` Parameter.

**Comment by chrostek** [comment]
Es geht nicht um einen Parameter - es geht (so wie ich es verstehe) darum als Value (für LEVEL) statt einer Zahl "OLD_LEVEL" ans Gerät zu senden.

**Reply by SukramJ** [reply]
Dann schau dir doch mal deine eigenen Links an, da sendet keiner `OLD_LEVEL` ans Gerät.

**Comment by chrostek** [comment]
okay, hast Recht - sorry und danke für die schnelle Rückmeldung.
